src/index.html

```diff
@@ -13,6 +14,26 @@
       <div class='vertical-line'> </div>
       <img class='logo-img' src='assets/tcg_logo.png' alt='logo'>
     </div>
+    
+    <div class='features-container'>
+      <div class='features-wrapper'>
+        <div class='feature-container'>
+          <img class='traking-img' src='assets/ic_list.png' alt='list'>
+          <p class='feature-title'>Keep track of your money</p>
```

Comment:
this should be a headline tag since it's the feature title, same for every feature title.